drf_spectacular/views.py

```diff
@@ -81,12 +82,23 @@ class SpectacularSwaggerView(APIView):
 
     @extend_schema(exclude=True)
     def get(self, request, *args, **kwargs):
+        swagger_bundle_js_href = "{}/swagger-ui-bundle.js".format(spectacular_settings.SWAGGER_UI_DIST)
+        swagger_bundle_css_href = "{}/swagger-ui.css".format(spectacular_settings.SWAGGER_UI_DIST)
+        swagger_bundle_favicon_href = "{}".format(spectacular_settings.FAVICON_HREF)
+
         schema_url = self.url or reverse(self.url_name, request=request)
         if request.GET.get('lang'):
             schema_url += f'{"&" if "?" in schema_url else "?"}lang={request.GET.get("lang")}'
-        settings = json.dumps(spectacular_settings.SWAGGER_UI_SETTINGS)
+        swagger_ui_settings = {'url': schema_url}
+        swagger_ui_settings.update(spectacular_settings.SWAGGER_UI_SETTINGS)
+
         return Response(
-            {'schema_url': schema_url, 'settings': settings},
+            {
```

Comment:
please restore the original version of this file and just add `dist` and `favicon_href` variable to the template. that should be sufficient.

```python
{
    'schema_url': schema_url, 
    'settings': settings,
    'dist': spectacular_settings.SWAGGER_UI_DIST,
    'favicon_href': spectacular_settings.SWAGGER_UI_FAVICON_HREF
},